src/data/script/ConstraintChangerModPlugin.java

```diff
@@ -159,6 +461,94 @@ private void handleAutomaticShips() {
+
+            // This method does the reflection part, which also doesn't quite work and is breaking the LunaListener
+//            handleFieldRepairsNonWorkingPart();
+
+            writeLunaSettingToRealSetting(FIELD_SHARED_FIELD_REPAIRS_AND_CONTAINMENT_PROCEDURES_OP_THRESHOLD);
+            writeLunaSettingToRealSetting(FIELD_FIELD_REPAIRS_MIN_HULL);
+            writeLunaSettingToRealSetting(FIELD_FIELD_REPAIRS_MAX_HULL);
+            writeLunaSettingToRealSetting(FIELD_FIELD_REPAIRS_MIN_CR);
+            writeLunaSettingToRealSetting(FIELD_FIELD_REPAIRS_MAX_CR);
+            writeLunaSettingToRealSetting(FIELD_FIELD_REPAIRS_REPAIR_RATE_BONUS);
+            writeLunaSettingToRealSetting(FIELD_FIELD_REPAIRS_INSTA_REPAIR_PERCENT);
+            logger.info("[SHARK] <---- handleFieldRepairs()");
+        }
+
+        private void handleFieldRepairsNonWorkingPart() {
+            logger.info("[SHARK] ----> handleFieldRepairsNonWorkingPart()");
+            try {
+                ReflectionUtils.INSTANCE.modifyFinalField("MIN_HULL", FieldRepairs.class, (float) safeUnboxing(LunaSettings.getInt(MOD_ID, FIELD_FIELD_REPAIRS_MIN_HULL)));
+                logger.info("[SHARK] \t handleFieldRepairs()\tcheckpoint 2");
+                ReflectionUtils.INSTANCE.modifyFinalField("MAX_HULL", FieldRepairs.class, (float) safeUnboxing(LunaSettings.getInt(MOD_ID, FIELD_FIELD_REPAIRS_MAX_HULL)));
+                logger.info("[SHARK] \t handleFieldRepairs()\tcheckpoint 3");
+                ReflectionUtils.INSTANCE.modifyFinalField("MIN_CR", FieldRepairs.class, (float) safeUnboxing(LunaSettings.getInt(MOD_ID, FIELD_FIELD_REPAIRS_MIN_CR)));
+                logger.info("[SHARK] \t handleFieldRepairs()\tcheckpoint 4");
+                ReflectionUtils.INSTANCE.modifyFinalField("MAX_CR", FieldRepairs.class, (float) safeUnboxing(LunaSettings.getInt(MOD_ID, FIELD_FIELD_REPAIRS_MAX_CR)));
+                logger.info("[SHARK] \t handleFieldRepairs()\tcheckpoint 5");
+                ReflectionUtils.INSTANCE.modifyFinalField("REPAIR_RATE_BONUS", FieldRepairs.class, (float) safeUnboxing(LunaSettings.getInt(MOD_ID, FIELD_FIELD_REPAIRS_REPAIR_RATE_BONUS)));
+                logger.info("[SHARK] \t handleFieldRepairs()\tcheckpoint 6");
+                ReflectionUtils.INSTANCE.modifyFinalField("INSTA_REPAIR_PERCENT", FieldRepairs.class, (float) safeUnboxing(LunaSettings.getInt(MOD_ID, FIELD_FIELD_REPAIRS_INSTA_REPAIR_PERCENT)));
+                logger.info("[SHARK] \t handleFieldRepairs()\tcheckpoint 7");
+            } catch (Exception ex) {
+                logger.error("[SHARK] \t handleFieldRepairs()\tcaught exception "+ex);
+                logger.error(StacktraceUtils.unwindStacktrace(ex.getStackTrace()));
+            }
+            logger.info("[SHARK] <---- handleFieldRepairsNonWorkingPart()()");
+        }
```

Comment:
do we get rid of this since it doesn't work or let it sit around for a while longer?